src/public/modules/forms/config/forms.client.routes.js

```diff
@@ -36,14 +36,15 @@ angular.module('forms').config([
           FormData: [
             'FormApi',
             '$transition$',
-            function (FormApi, $transition$) {
-              return FormApi.preview($transition$.params()).$promise.then(
-                (FormData) => {
+            '$q',
+            function (FormApi, $transition$, $q) {
+              return $q
+                .when(FormApi.previewForm($transition$.params().formId))
```

Comment:
same comment as previous review - `resolve` expects a Promise, so this doesn't need to be wrapped in `$q.when`